generators/app/templates/_tslint.json

```diff
@@ -148,7 +148,7 @@
     "component-selector": [
       true,
       "element",
-      "app",
+      ["app", "i18n"],
```

Comment:
the `app` prefix means only used in app.  `i18n` means  not depend on app.

Reply:
In this case I believe that if you remove i18n, it works exactly the same way (because i18n is a sub-folder of app). So that would be unnecessary code.